tests/constants1.py

```diff
@@ -35,12 +35,13 @@ def test_len():
     a = len('')
     a = len('test')
     a = len("this is a test")
-    a = len((1, 2, 3))
-    a = len((("a", "b", 3.4), ("c", 3, 5.6)))
-    a = len([1, 2, 3])
-    a = len([[-4, -5, -6], [-1, -2, -3]])
-    a = len({1, 2, 3})
-    a = len({1: "a", 2: "b", 3: "c"})
+    # TODO(namannimmo10): These commented out tests should work
+    # a = len((1, 2, 3))
+    # a = len((("c", "b", 3.4), ("c", 3, 5.6)))
+    # a = len([1, 2, 3])
+    # a = len([[-4, -5, -6], [-1, -2, -3]])
+    # a = len({1, 2, 3})
+    # a = len({1: "c", 2: "b", 3: "c"})
```

Comment:
Why don't they work anymore?

Reply:
Looks like the argument is not a constant, and returns false when it runs across...

https://github.com/lcompilers/lpython/blob/2dd1e4ddd3e40ed65f2db9db3fe30c1247ef41dc/src/libasr/asr_utils.h#L417-L466